EXCLUSION.md

```diff
@@ -1,3 +1,44 @@
-TwoFactorAuth.org Blacklist
-=================
+# TwoFactorAuth.org excluded categories and websites
 
+Below is a list of categories and websites that we've opted out from listing on TwoFactorAuth.org.
```

Comment:
```suggestion
Below is a list of categories and websites that we, [the collaborators](https://github.com/orgs/2factorauth/teams/collaborators/members), have opted to not list list on TwoFactorAuth.org.
```

Reply:
collaborators link is 404.
`list` word is listed listed twice.